src/main/java/org/intellij/sdk/codesync/actions/FilePlaybackAction.java

```diff
@@ -78,11 +81,15 @@ public void actionPerformed(@NotNull AnActionEvent e) {
 
         if (configRepo == null) {
             NotificationManager.notifyError(
-                    "An error occurred trying to perform file playback action.", project
+                String.format(
```

Comment:
It think If repo is not synced, View file playback Or repo playback options should not be visible

Reply:
Yes, that is the case as of now as well. This line here is just for completeness. Ideally this should never happen.

ref: https://github.com/codesyncapp/codesync-intellij/blob/master/src/main/java/org/intellij/sdk/codesync/actions/RepoPlaybackAction.java#L25

Reply:
Sounds good. Thanks